File: tools/src/main/java/com/example/alexeyglushkov/tools/ExceptionTools.java
package com.example.alexeyglushkov.tools;

/**
 * Created by alexeyglushkov on 24.02.18.
 */

public class ExceptionTools {
    public static void throwIfNull(Object obj, String message) throws NullPointerException {
        if(obj == null) {
            throw new NullPointerException(message);
        }
    }
}

File: tools/src/main/java/com/example/alexeyglushkov/tools/ExceptionTools.kt
package com.example.alexeyglushkov.tools

/**
 * Created by alexeyglushkov on 24.02.18.
 */
object ExceptionTools {
    @JvmStatic
    @Throws(NullPointerException::class)
    fun throwIfNull(obj: Any?, message: String?) {
        if (obj == null) {
            throw NullPointerException(message)
        }
    }
}